package.json

```diff
@@ -17,7 +17,7 @@
   },
   "license": "MIT",
   "devDependencies": {
-    "koa": "^0.21.0",
+    "koa": "^2.0.0",
```

Comment:
That would effectively install `^2.2.0` as of today, so there's no real danger in getting a broken release. But, yes - you could explicitly exclude `2.0.0` if you like.